.github/workflows/ci-linux-on-main-pr.yml

```diff
@@ -26,14 +26,13 @@ jobs:
         run: |
           sudo apt-get update -y
           sudo apt-get install -y 
-          sudo apt-get remove -y llvm-12
+          sudo apt-get remove -y llvm-12 llvm-11 clang-format-11 clang-tidy-11 
           sudo apt-get install -y curl pkg-config findutils  wget
-          sudo apt install -y clang-11 cmake clang-format-11 clang-tidy-11 
-          sudo apt-get install -y llvm-11 lldb-11 llvm-11-dev libllvm11 llvm-11-runtime opt 
+          sudo apt install -y clang-13 cmake clang-format-13 clang-tidy-13 
+          sudo apt-get install -y llvm-13 lldb-13 llvm-13-dev libllvm11 llvm-13-runtime opt 
           sudo apt-get install -y nodejs npm
           sudo apt install -y python3 python3-pip 
-          sudo apt-get remove -y llvm-10
```

Comment:
(nit) In some files the line `sudo apt-get remove -y llvm-10` is removed, but in the others it is not. It is suspicious.

Reply:
Thanks, this has been updated.

Reply:
FYI: Still in `.github/workflows/trusted-ci-linux-on-main-pr.disabled` L50.